backend/app/library/error/message.go

```diff
@@ -0,0 +1,38 @@
+package error
+
+import (
+	"os"
+
+	"github.com/jumpei00/board/backend/app/library/logger"
+	"gopkg.in/yaml.v3"
+)
+
+var messageYaml *errCause
+
+func init() {
+	file, err := os.ReadFile("./message.yaml")
+	if err != nil {
+		logger.Fatal("no open message.yaml")
+	}
+
+	errContents := &errCause{}
+	yaml.Unmarshal(file, errContents)
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Error return value of `yaml.Unmarshal` is not checked (errcheck)